ZhaoYuLin/Dog.java

```diff
@@ -6,19 +6,50 @@
     int age;
     boolean rabies;
 
-    public Dog(String name, String type, int age) {
+    public Dog(String name,String type,int age) {
         this.name = name;
         breed = type;
         this.age = age;
         rabies = false;
     }
-   
+
     public void bark() {
         System.out.println("bork");
     }
-    
+
     public static void main(String[] args) {
```

Comment:
We usually try to keep all the methods (not including the main method) together. Either move the main method below all the other methods (preferred) or move it above them, but after the constructors.